### PR TITLE
GFM-71 - Converted country typeahead to ES6

### PR DESCRIPTION
**Pull request by gxxm**

### CODE REVIEW

**Comment by JoeChapman** [comment]
@gxxm I'm not sure about this. I feel like the typeahead is worth creating as a module.

**Comment by easternbloc** [comment]
Erm you realise this is on the frontend right. You don't have any babel for this. It's not gonna work. Don't do this.

**Comment by JoeChapman** [comment]
@easternbloc makes a good point. My bad

**Comment by joefitter** [comment]
we have a `browserify` build step - we should add babelify transform here if we want to do this - I mentioned this a while back. Perhaps it would be better to go with @JoeChapman 's suggestion of extracting to a module.

**Comment by joefitter** [comment]
let me rephrase ^ it __would__ be better to extract this to a module 😄
